src/main/java/io/cdap/e2e/pages/actions/CdfGcsActions.java

```diff
@@ -73,4 +70,43 @@ public static void skipHeader() {
   public static void getSchema() {
     CdfGCSLocators.getSchemaButton.click();
   }
+
+  public static void delimiter() throws IOException {
+    CdfGCSLocators.delimiter.sendKeys(SeleniumHelper.readParameters("delimiter_@"));
+  }
+
+  public static void selectFormat(String formatType) throws IOException, InterruptedException {
+    CdfGCSLocators.format.click();
+    String format = SeleniumHelper.readParameters(formatType);
+    SeleniumHelper.waitAndClick(
+      SeleniumDriver.getDriver().findElement(By.xpath("//*[contains(text(),'" + format + "')]")));
+  }
+
+  public static void schemaValidation() {
+    List<WebElement> elements = SeleniumDriver.getDriver().findElements(By.xpath("//*[@placeholder='Field name']"));
+    for (WebElement schema : elements) {
+      String actualStr = schema.getAttribute("value");
+      String expectedStr = "offset";
+      if (expectedStr.equalsIgnoreCase(actualStr)) {
+        checkParameter = false;
+      } else {
+        checkParameter = true;
+      }
+    }
+    assertionVerification();
+  }
+
+  private static void assertionVerification() {
```

Comment:
no need for this function - just call the assert directly. 
Nit: rename schemaValidation() -> validateSchema()

Reply:
done